sim/druid/item_sets_pve.go

```diff
@@ -485,12 +490,80 @@ var ItemSetFuryOfStormrage = core.NewItemSet(core.ItemSet{
 	Bonuses: map[int32]core.ApplyEffect{
 		// Swipe(Bear) also causes your Maul to hit 1 additional target for the next 6 sec.
 		2: func(agent core.Agent) {
+			druid := agent.(DruidAgent).GetDruid()
+
+			if druid.Env.GetNumTargets() == 1 {
+				return
+			}
+
+			var curDmg float64
+
+			cleaveHit := druid.RegisterSpell(Bear, core.SpellConfig{
+				ActionID:    core.ActionID{SpellID: 467217},
+				SpellSchool: core.SpellSchoolPhysical,
+				ProcMask:    core.ProcMaskEmpty,
+				Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagNoOnCastComplete | core.SpellFlagPassiveSpell,
+
+				DamageMultiplier: 1,
+				ThreatMultiplier: 1,
+
+				ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+					spell.CalcAndDealDamage(sim, target, curDmg, spell.OutcomeAlwaysHit)
+				},
+			})
```

Comment:
It looks like this spell modifier the `jump target` of Maul which means it just causes Maul to hit a second target, so I think you just want to cast Maul at the second target rather than creating a brand new damaging spell